circle/circle.go

```diff
@@ -18,6 +18,15 @@ Square: 201.06192982974676
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
 	// ++ Your code here! ++
+	fmt.Println("Enter the radius:")
+	var rds float64
+	fmt.Scan(&rds)
+	var Pi float64 = 3.14159265358979323846264338327950288419716939937510582097494459
+	var prm = 2 * (Pi * rds)
+	var sqr = Pi * (rds * rds)
+	fmt.Printf("Perimeter: %v\n", prm)
```

Comment:
nit: Please avoid using `fmt.Printf` for now, since we haven't discussed it yet. Go for `fmt.Println("Perimeter:", p)` atm. Otherwise please explain what is `%v` here, and why it isn't `%f`? What would be the difference?

Reply:
Good and impressive! Feel free to keep `fmt.Printf` here and use it in the future. I would recommend you learned about other arguments (ask myself or Pavel if smth isn't clear).